spec/javascripts/helpers/jasmine-jquery.js

```diff
@@ -0,0 +1,841 @@
+/*!
```

Comment:
This jasmine-jquery library has already been merged! https://github.com/julianguyen/ifme/commit/1bcb1bf21ccfef9e44d74e015ef791b52cfbd2d2